### PR TITLE
[INLONG-6174][Sort] MySql connector support meta data with debezium format 

### LINKED ISSUE #6174: [Feature][Sort] MySQL connector support meta data with debezium format
### Description

[Feature] Mysql connector support meta data with debezium format 

### Use case

[Feature] Mysql connector support meta data with debezium format 

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by EMsnap**
- Fixes #6174 

### Motivation

[INLONG-6174][Sort] Mysql connector support meta data with debezium format 

### Modifications

[INLONG-6174][Sort] Mysql connector support meta data with debezium format 

### Verifying this change

run allmigrateUt

### CODE REVIEW

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/KafkaLoadNode.java [hunk 1]
```diff
@@ -226,8 +226,8 @@ public String getMetadataKey(MetaField metaField) {
             case OP_TYPE:
                 metadataKey = "value.op-type";
                 break;
-            case DATA:
-                metadataKey = "value.data";
+            case DATA_CANAL:
```
Comment (yunqingmoswu) [inline inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/KafkaLoadNode.java:229]: Maybe it is better to add 'source.multiple.format' to decide which serialization to use？

Comment (EMsnap) [reply inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/KafkaLoadNode.java:229]: I guess using two seperated meta data is fare enough?

File: inlong-common/src/main/java/org/apache/inlong/common/enums/MetaField.java [hunk 2]
```diff
@@ -62,12 +62,22 @@ public enum MetaField {
     /**
      * Represents a canal json of a record in database (in string format)
      */
-    DATA,
```
Comment (yunqingmoswu) [inline inlong-common/src/main/java/org/apache/inlong/common/enums/MetaField.java:65]: Please keep in mind code compatibility

Comment (EMsnap) [reply inlong-common/src/main/java/org/apache/inlong/common/enums/MetaField.java:65]: done thanks

File: inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/table/MySqlReadableMetadata.java [hunk 3]
```diff
@@ -98,7 +100,7 @@ public Object read(SourceRecord record) {
             }),
 
     DATA(
-            "meta.data",
```
Comment (yunqingmoswu) [inline inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/table/MySqlReadableMetadata.java:101]: Please keep in mind code compatibility.

Comment (EMsnap) [reply inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/table/MySqlReadableMetadata.java:101]: done

**Review by yunqingmoswu — APPROVED** [review]
LGTM

File: inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/table/MySqlReadableMetadata.java [hunk 4]
```diff
@@ -379,6 +415,38 @@ public Object read(SourceRecord record) {
...
+        dataList.add(field);
+
+        CanalJson canalJson = CanalJson.builder()
+            .data(dataList).database(databaseName)
+            .sql("").es(opTs).isDdl(false).pkNames(getPkNames(tableSchema))
+            .mysqlType(getMysqlType(tableSchema)).table(tableName).ts(ts)
+            .type(getOpType(record)).sqlType(getSqlType(tableSchema)).build();
+
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
```
Comment (healchow) [inline inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/table/MySqlReadableMetadata.java:443]: Suggest using a global static instance of ObjectMapper.

Comment (EMsnap) [reply inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/table/MySqlReadableMetadata.java]: done thanks

**Review by healchow — APPROVED** [review]

**Review by gong — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**